packages/esm-appointments-app/src/hooks/useAppointmentList.tsx

```diff
@@ -67,3 +67,26 @@ export const useEarlyAppointmentList = (startDate?: string) => {
   }));
   return { earlyAppointmentList: (appointments as Array<any>) ?? [], isLoading, error };
 };
+
+export const useCompletedAppointmentList = (startDate?: string) => {
+  const appointmentDate = useAppointmentDate();
+  const forDate = startDate ? startDate : appointmentDate;
+  const url = `/ws/rest/v1/appointment/completedAppointment?forDate=${forDate}`;
```

Comment:
Great job on the code! To further optimize it, we suggest updating the endpoint URL to reuse what already exists. This would simplify the code and reduce network overhead. We recommend using the following URL format to retrieve all appointments with the specified appointmentStatus, including those that have already been marked as completed: `/ws/rest/v1/appointment/appointmentStatus?status=${appointmentStatus}&forDate=${forDate}`

Reply:
Thanks Donald however the logic for this appointment cohort is different and i'ts not checking the appointment status, that's why we have it in a different endpoint